java/configuration/src/main/java/sleeper/configuration/properties/instance/CdkDefinedInstanceProperty.java

```diff
@@ -124,7 +124,22 @@ public interface CdkDefinedInstanceProperty extends InstanceProperty {
             .description("The name of the table responsible for tracking query progress.")
             .propertyGroup(InstancePropertyGroup.QUERY)
             .build();
-
+    CdkDefinedInstanceProperty LEAF_PARTITION_QUERY_QUEUE_URL = Index.propertyBuilder("sleeper.leaf.partition.query.queue.url")
+            .description("The URL of the queue responsible for sending a leaf partition query to sleeper.")
+            .propertyGroup(InstancePropertyGroup.QUERY)
+            .build();
+    CdkDefinedInstanceProperty LEAF_PARTITION_QUERY_QUEUE_ARN = Index.propertyBuilder("sleeper.leaf.partition.query.queue.arn")
+            .description("The ARN of the queue responsible for sending a leaf partition query to sleeper.")
+            .propertyGroup(InstancePropertyGroup.QUERY)
+            .build();
+    CdkDefinedInstanceProperty LEAF_PARTITION_QUERY_QUEUE_DLQ_URL = Index.propertyBuilder("sleeper.leaf.partition.query.dlq.url")
+            .description("The URL of the dead letter queue used when leaf partition querying sleeper.")
+            .propertyGroup(InstancePropertyGroup.QUERY)
+            .build();
+    CdkDefinedInstanceProperty LEAF_PARTITION_QUERY_QUEUE_DLQ_ARN = Index.propertyBuilder("sleeper.leaf.partition.query.dlq.arn")
```

Comment:
These properties should start with sleeper.query to be consistent with the other properties.

Reply:
Updated